CMakeLists.txt

```diff
@@ -20,7 +20,7 @@ set(DESCRIPTION_TRANSLATIONS
 
 本輸入法也同時支援帶調漢語拼音輸入。")
 
-set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall")
+set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Wall -Wextra -Werror")
```

Comment:
Could you consider limit `-Werror` to CI only instead of using it by default? ibus-chewing is a popular open-source project, which may be built using different compiler versions on several Linux distributions other than the ones tested in CI. As a result, package maintainers for Linux distributions may still get failures caused by `-Werror` when everything works in CI.

For example: (not tested)
```CMake
option(Werror "Treat warnings as errors" OFF)
if(Werror)
  set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -Werror")
endif()
```

Reply:
Sure, thanks for this idea! :D

Reply:
Thank you very much!